### PR TITLE
Add active effect to navbar buttons

### LINKED ISSUE #9: Active Effect to Navbar-Buttons
### Value description/ definition
As a user
I want to see in the Navbar on wich page I am actually 
so that I exactly know on one look, on wich page I am

### Description
Screenshot 2022-05-27 at 16 32 03

### Acceptance criteria
- [x] The buttons must change their color when the right page is showing

### Tasks
- [x] Create a branch
- [x] Make first Commit for Upstream
- [x] Create functionality on the buttons for when I have switched the page

### Complexity
medium

### PR DESCRIPTION
**Pull request by Chaosfuchs**
Add active effect to the buttons on the Navbar, when the page is shown

### CODE REVIEW

File: components/Header.js [hunk 1]
```diff
@@ -1,8 +1,11 @@
...
+import styled, { css } from 'styled-components';
 
 export const Title = styled.h1`
-  background-color: ${({ theme }) => theme.colors.background};
-  color: ${({ theme }) => theme.colors.primary};
+  ${({ theme }) => css`
+    background-color: ${theme.colors.background};
+    color: ${theme.colors.primary};
+  `}
+  box-shadow: 5px 5px 20px black;
```
Comment (CJAkkuec) [inline components/Header.js]: Is there a reason why you're not using your box-shadow variable here?

File: components/Navbar.js [hunk 2]
```diff
@@ -1,18 +1,29 @@
...
 
 export default function Navbar() {
+  const router = useRouter();
+
   return (
     <>
       <StyledNavbar>
         <Link passHref href="/">
-          <StyledButtonNavbar type="radio">Home</StyledButtonNavbar>
+          <StyledButtonNavbar isActive={router.pathname === '/'}>
```
Comment (TheMarv) [inline components/Navbar.js:12]: In this case router.pathname is viable. But if this would be a dynamic router with [id] for example, you would need to use asPath. This is just an advice for the future, no need for changes.

**Review by CJAkkuec — COMMENTED** [review]
Other than my minor note about the box-shadow, the PR looks good to me! :) Feel free to proceed.

File: pages/index.js [hunk 3]
```diff
@@ -6,10 +6,11 @@ import Link from 'next/link';
 export default function Home() {
   return (
     <>
-      <Title>Welcome to RPG 4-you</Title>
+      <Title>RPG Sheet-Creator</Title>
```
Comment (TheMarv) [inline pages/index.js:9]: try not to change code that is not related to the pull request. In this case you wanted to add a navigation but edited something on the main page, which is not related.

**Review by TheMarv — APPROVED** [review]
looks good to me!
